src/lib/form/modelFormSchemas.ts

```diff
@@ -37,12 +37,15 @@ const style = z.object({
     icon: z.string().optional(),
 })
 
+const dateWithoutTimestampSchema = z.coerce.date()
```

Comment:
To be pedantic, this is now just a date schema, right? It doesn't actually do what the variable name says. In that case we would want something like:

```
const dateWithoutTimestampSchema = z
            .string()
            .transform((v) => v.substring(0, 10))
            .pipe(z.string().date()),
```